src/applications/letters/actions/letters.js

```diff
@@ -74,7 +74,11 @@ export function getLetterList(dispatch) {
       } else {
         dispatch({ type: GET_LETTERS_FAILURE });
       }
-      throw new Error(`vets_letters_error_getLetterList: ${status}`);
+      const lettersError = new Error('vets_letters_error_getLetterList');
+      Raven.captureException(lettersError, {
+        fingerprint: ['{{ default }}', status],
+      });
+      return Promise.reject();
```

Comment:
Added this in here because `getLetterListAndBLSOptions` has a catch callback expecting this, whereas before we were throwing an error instead of using `captureException`